### ISSUE TITLE
Simplify API for querying for Accounts

### ISSUE THREAD

**Issue by vestrel00** [vestrel00/contacts-android#153]
While working on #147 in preparation for #144 and #145, I noticed that the `AccountsQuery` can be simplified to adhere to the pattern followed by the other APIs in the library.

The API can be simplified to,

```kotlin
val result = accountsQuery
      .withTypes("com.google", "com.hotmail")
      .associatedWith(rawContact1, rawContact2)
      .find()
```

- [x] Update https://github.com/vestrel00/contacts-android/blob/main/howto/howto-query-accounts.md
- [x] Update README
- [x] Update blogs

**Comment by vestrel00** [comment]
This will be closed in scope of #147

**Comment by vestrel00** [comment]
Included in release; https://github.com/vestrel00/contacts-android/releases/tag/0.1.10
